financial-connections/src/main/java/com/stripe/android/financialconnections/features/institutionpicker/InstitutionPickerScreen.kt

```diff
@@ -536,15 +560,17 @@ internal fun InstitutionPickerPreview(
 ) {
     FinancialConnectionsPreview {
         InstitutionPickerContent(
-            previewText = state.previewText,
             payload = state.payload,
-            institutionsProvider = { state.searchInstitutions },
+            institutions = state.searchInstitutions,
             searchMode = state.searchMode,
+            previewText = state.previewText,
             onQueryChanged = {},
```

Comment:
I would should we provide default values here if we have a lot of empty blocks

Reply:
This is just for Compose Previews, I'd rather not use default values to prevent forgetting them on their real usage : )